src/plugins/discover/public/application/components/default_discover_table/table_header_column.tsx

```diff
@@ -13,24 +13,21 @@ import './_table_header.scss';
 
 import React, { ReactNode } from 'react';
 import { i18n } from '@osd/i18n';
-import { EuiButtonIcon, EuiDataGridSorting, EuiToolTip } from '@elastic/eui';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import { SortOrder } from './helper';
 
 interface Props {
-  currentIdx: number;
   colLeftIdx: number; // idx of the column to the left, -1 if moving is not possible
   colRightIdx: number; // idx of the column to the right, -1 if moving is not possible
   displayName: ReactNode;
   isRemoveable: boolean;
   isSortable?: boolean;
   name: string;
-  onChangeSortOrder?: (cols: EuiDataGridSorting['columns']) => void;
+  onChangeSortOrder?: (sortOrder: SortOrder[]) => void;
   onMoveColumn?: (name: string, idx: number) => void;
-  onReorderColumn?: (col: string, source: number, destination: number) => void;
+  onReorderColumn?: (colName: string, destination: number) => void;
```

Comment:
Why do we have `onMove` and `onReorder`?

Reply:
remove this